src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -0,0 +1,208 @@
+              maxChars={2500}
+              id="outcome"
+            />
+            <Field
+              name="subjectPrimary"
+              component={RenderSelectField}
+              label={<strong>Primary Subject: *</strong>}
+              options={subjectOptions}
+              required
+            />
+            <Field
+              name="subjectSecondary"
+              component={RenderSelectField}
+              label={<strong>Secondary Subject:</strong>}
+              options={subjectOptions}
+            />
+            <Field
+              name="subjectTertiary"
+              component={RenderSelectField}
+              label={<strong>Tertiary Subject:</strong>}
+              options={subjectOptions}
+            />
+            <Field
+              name="imageSrc"
+              component={ImageUpload}
+              label={<strong>Course Image: *</strong>}
+              id="image"
+            />
+            <Field
+              name="prerequisites_raw"
+              component={RichEditor}
+              label={<strong>Prerequisites:</strong>}
+              maxChars={1000}
+              id="prereq"
+            />
+            <Field
+              name="level_type"
+              component={RenderSelectField}
+              label={<strong>Course Level: *</strong>}
+              options={levelTypeOptions}
+              required
+            />
+            <Field
+              name="learner_testimonials"
+              component={RichEditor}
+              label={<strong>Learner Testimonials:</strong>}
+              maxChars={500}
+              id="learner-testimonials"
+            />
+            <Field
+              name="faq"
+              component={RichEditor}
+              label={<strong>Frequently Asked Questions:</strong>}
+              maxChars={2500}
+              id="faq"
+            />
+            <Field
+              name="additional_information"
+              component={RichEditor}
+              label={<strong>Additional Information:</strong>}
+              maxChars={2500}
+              id="additional-information"
+            />
+            <Field
+              name="syllabus_raw"
+              component={RichEditor}
+              label={<strong>Syllabus:</strong>}
+              maxChars={500}
+              id="syllabus"
+            />
+            <Field
+              name="videoSrc"
+              component={RenderInputTextField}
+              type="text"
+              label={<strong>About Video Link: *</strong>}
+              required
+            />
+            <Field
+              name="mode"
+              component={RenderInputTextField}
+              type="text"
+              label={<strong>Course Mode:</strong>}
+              disabled
+            />
+            {price && <Field
+              name="price"
+              component={RenderInputTextField}
+              type="number"
+              label={<strong>Course Price: *</strong>}
+            />}
```

Comment:
This should use some sort of "price" formatting. So "2.50" shows instead of "2.5" - is that easy?

Reply:
Some of the logic from [prospectus formatting](https://github.com/edx/prospectus/blob/master/src/utils/formatting.js#L5-L19) might be helpful here.  The tests [here](https://github.com/edx/prospectus/blob/master/src/utils/formatting.test.js) give some examples of how it works